pkgs/data/themes/levuaska-gtk-theme/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, gtk_engines }:
+
+stdenv.mkDerivation {
+  version = "2021-1-31";
+  pname = "levuaska-gtk-theme";
+
+  src = fetchFromGitHub {
+    owner = "selene57";
```

I think you put the wrong owner.
```suggestion
    owner = "saimoomedits";
```

---

The owner is correct. The theme comes from saimoomedit's dotfiles under their .themes folder. However, none of the required scripts or other files to actually allow for a package install are in the repo. With permission from that user (and implicit permission from the licensing of the gtk theme he based them on), I created a repo and added the necessary auxiliary files to allow automake to work. As I'm the owner of the repo, I have to be listed for that parameter. I have made sure to specify both in that repo and in the default.nix file for the package that saimoomedits is the actual owner/creator of the underlying theme though.

I hope that clears it up?

---

Oh, okay. I seem to have misspelled your name when looking for your fork.